vue/readme.md

```diff
@@ -194,37 +197,37 @@ Variate assigns each visitor an `experimentBucket`. This is a random number from
 
 ```json
 "variations": [
-                    {
-                        "id": 1,
-                        "traffic_allocation": {
-                            "max": 50,
-                            "min": 1
-                        },
-                        "components": {
-                            "HomeHero": {
-                                "id": 1,
-                                "attributes": {
-                                    "headline": "The A/B testing tool for the modern web"
-                                }
-                            }
-                        }
-                    },
-                    {
-                        "id": 2,
-                        "traffic_allocation": {
-                            "max": 100,
-                            "min": 51
-                        },
-                        "components": {
-                            "HomeHero": {
-                                "id": 1,
-                                "attributes": {
-                                    "headline": "The A/B testing tool that marketers and developers can agree on"
-                                }
-                            }
-                        }
-                    }
-                ]
+    {
+        "id": 1,
+        "traffic_allocation": {
```

Comment:
@alhankeser Can you update according to https://github.com/VariateApp/variate-config/pull/1 please?